exporter/awsemfexporter/metric_translator.go

```diff
@@ -116,12 +124,73 @@ func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, dimensionRollupOption stri
 				continue
 			}
 			cwMetricList := getMeasurements(&metric, namespace, OTLib, dimensionRollupOption)
+			batchCWMetrics(cwMetricList, groupedCWMetricMap, cwMetricsMap)
 			cwMetricLists = append(cwMetricLists, cwMetricList...)
 		}
 	}
 	return cwMetricLists, totalDroppedMetrics
 }
 
+func batchCWMetrics(cwMetricList []*CWMetrics, groupedCWMetricMap map[string]*GroupedCWMetric, cwMetricsMap map[string]*CWMetrics) {
+	for _, i := range cwMetricList {
```

Comment:
nit: could we choose a better name than `i`? In this case, would `cwMetric` or `metric` be better?